components/channel_header/channel_header.js

```diff
@@ -227,6 +232,13 @@ export default class ChannelHeader extends React.PureComponent {
         }
     }
 
+    closeQuickSwitchModalIfOpen = () => {
+        const {isQuickSwitcherOpen} = this.props;
+        if (isQuickSwitcherOpen) {
```

Comment:
It /seems/ weird to have to check if we can close the modal -- is `closeModal` perhaps already a no-op if the modal isn't open?